mm-bot/src/services/overall_fee_calculator.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+
+from web3 import Web3
+
+from models.order import Order
+from services.ethereum import create_transfer, estimate_transaction_fee, get_gas_price
+from services.withdrawer.ethereum_withdrawer import EthereumWithdrawer
+
+
+async def estimate_overall_fee(order: Order) -> int:
+    transfer_fee = await asyncio.to_thread(estimate_transfer_fee, order)
+    message_fee = await estimate_message_fee(order)
+    withdraw_fee = estimate_withdraw_fee()
+    return transfer_fee + message_fee + withdraw_fee
+
+
+def estimate_transfer_fee(order: Order) -> int:
+    dst_addr_bytes = int(order.recipient_address, 0)
+    deposit_id = Web3.to_int(order.order_id)
+    amount = Web3.to_int(order.get_int_amount())
+
+    unsent_tx, signed_tx = create_transfer(deposit_id, dst_addr_bytes, amount)
+
+    return estimate_transaction_fee(unsent_tx)
+
+
+def estimate_withdraw_fee() -> int:
```

Comment:
```suggestion
def estimate_yab_withdraw_fee() -> int:
```
it's not clear if it's the withdraw from starkgate or our function withdraw in the yabtransfer contract

Reply:
Done